packages/Button/stories/Button.stories.mdx

```diff
@@ -0,0 +1,19 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+import { Description } from "@storybook/addon-docs/blocks";
+import { getStoryName } from "storybook/storyTree";
+import Readme from "../README.md";
+import MDX from "./Button.mdx";
+import Button from "../src/";
+
+export const params = {
+  options: {
+    isToolshown: true,
+    showPanel: false,
+  },
+};
+
+<Meta title={`${getStoryName("Button")}/Docs`} component={Button} parameters={params} />
+
+<Description markdown={Readme} />
+
+<MDX />
```

Comment:
Since this `Button.stories.mdx` file is now so small, how about we just add the content of `Button.mdx` right here to simplify things?